### PR TITLE
feat: replace nodejieba with @node-rs/jieba

### PR DESCRIPTION
**Pull request by F3n67u**
close https://github.com/easyops-cn/docusaurus-search-local/issues/140

### CODE REVIEW

**Comment by weareoutman** [comment]
Thanks, that was quick 👍

**Comment by F3n67u** [comment]
> Thanks, that was quick 👍

the test is failing. I am still working on this pr.

**Comment by F3n67u** [comment]
> > Thanks, that was quick 👍
> 
> the test is failing. I am still working on this pr.

@weareoutman Test is passed. ready to review.

**Comment by F3n67u** [comment]
@weareoutman could we add jieba as a dependency?

**Comment by weareoutman** [comment]
> @weareoutman could we add @node-rs/jieba as a dependency?

I think we could, if this library has no issue of slow download.

File: src/server/utils/tokenizer.ts [hunk 1]
```diff
@@ -1,13 +1,11 @@
 import lunr from "lunr";
-import nodejieba from "nodejieba";
+import jieba from "@node-rs/jieba";
 import { MatchMetadata } from "../../shared/interfaces";
 import { cutWordByUnderscore } from "./cutWordByUnderscore";
 
 // https://zhuanlan.zhihu.com/p/33335629
 const RegExpConsecutiveWord = /\w+|\p{Unified_Ideograph}+/u;
 
-nodejieba.load();
```
Comment (weareoutman) [inline src/server/utils/tokenizer.ts:9]: Don't we need to call `jieba.load()` according to [its document](https://github.com/napi-rs/node-rs/tree/main/packages/jieba#usage)?

Comment (F3n67u) [reply src/server/utils/tokenizer.ts:9]: not according to it's document, but I figure out in code, see:

https://github.com/napi-rs/node-rs/blob/3d329ba0ec01daabe4b9f6b1f2757d19d30be1c8/packages/jieba/src/lib.rs#L55

I encounter a test error "Jieba was loaded, could not load again" after I add the `load()`, but the error disappears after I remove the `load`;

**Comment by F3n67u** [comment]
> > @weareoutman could we add @node-rs/jieba as a dependency?
> 
> I think we could, if this library has no issue of slow download.

I include @node-rs/jieba as a dependency so that our user don't need include @node-rs/jieba as denpendency

**Review by weareoutman — APPROVED** [review]

**Pull request merged**